### ISSUE TITLE
Vulkan client support?

### ISSUE THREAD

**Issue by wyattwheeler** [deboogerxyz/ah4#12]
Feature / Addition suggestion:
Just wondering if it is possible to have the cheat support the Vulkan renderer for CS:GO? I am yet to tackle this myself, as I've got other projects going on.

I haven't found any concrete details as too why this couldn't be possible, not a biggie though.

**Comment by deboogerxyz** [comment]
Hi! Yes, it is possible. All we need to do is make a Vulkan+SDL2 backend for Nuklear. I have no experience using the Vulkan API, but there already is a project which uses Vulkan+GLFW. https://github.com/m0ppers/nuklear-glfw-vulkan. We'd have to adapt this code to use SDL2 instead of GLFW. Then the cheat would detect if you are using Vulkan by trying to dlopen it's dynamic library.

**Comment by oknyshuk** [comment]
hey @deboogerxyz, any progress on this? if I inject this in it's current state, I lose control in the game

**Comment by deboogerxyz** [comment]
No, Vulkan is not supported. I don't plan on updating this project as I don't play CS:GO anymore.
